Makefile

```diff
@@ -124,24 +123,27 @@ cluster-clean: $(KUBECTL)
 	$(KUBECTL) delete --ignore-not-found -f deploy/
 	$(KUBECTL) delete --ignore-not-found -f deploy/crds/nmstate_v1alpha1_nodenetworkstate_crd.yaml
 	$(KUBECTL) delete --ignore-not-found -f deploy/crds/nmstate_v1alpha1_nodenetworkconfigurationpolicy_crd.yaml
-	if [[ "$$KUBEVIRT_PROVIDER" =~ ^os-.*$$ ]]; then \
+	if [[ ! "$$KUBEVIRT_PROVIDER" =~ ^k8s-.*$$ ]]; then \
 		$(KUBECTL) delete --ignore-not-found -f deploy/openshift/; \
 	fi
 
 cluster-sync-resources: $(KUBECTL)
 	for resource in $(resources); do \
 		$(KUBECTL) apply -f $$resource || exit 1; \
 	done
-	if [[ "$$KUBEVIRT_PROVIDER" =~ ^os-.*$$ ]]; then \
+	if [[ ! "$$KUBEVIRT_PROVIDER" =~ ^k8s-.*$$ ]]; then \
 		$(KUBECTL) apply -f deploy/openshift/; \
 	fi
 
 cluster-sync-handler: cluster-sync-resources $(local_handler_manifest)
-	IMAGE_REGISTRY=localhost:$(shell $(CLI) ports registry | tr -d '\r') \
-				   make push-handler
-	# Temporary until image is updated with provisioner that sets this field
-	# This field is required by buildah tool
-	$(SSH) node01 'sudo sysctl -w user.max_user_namespaces=1024'
+	if [[ "$$KUBEVIRT_PROVIDER" =~ ^okd-.*$$ ]]; then \
+		IMAGE_REGISTRY=localhost:$(shell $(CLI) ports --prefix=$$KUBEVIRT_PROVIDER --container-name=cluster registry | tr -d '\r') \
```

Comment:
Is this a workaround of a known issue?

Reply:
It was failing on okd, and at kubernetes-nmstate we don't use buildah.

Reply:
How is it related to buildah?

Reply:
No clue, it was there at kubevirtci dind't do the research.